src/colvarbias.cpp

```diff
@@ -115,6 +117,22 @@ int colvarbias::init(std::string const &conf)
     return COLVARS_ERROR;
   }
 
+  // Use the scaling factors from a grid?
+  get_keyval(conf, "scaledBiasingForce", b_scaled_biasing_force, b_scaled_biasing_force);
+  if (b_scaled_biasing_force) {
+    cvm::log("scaledBiasingForce is set to on");
```

Comment:
The value of the flag would be already printed from `get_keyval()`.

Reply:
The redundant log message is removed.